packages/placement-ordering/configure/src/main.jsx

```diff
@@ -65,16 +65,17 @@ class Main extends React.Component {
       <div>
         <div className={classes.tabBar}>
           <Tabs onChange={this.onTabIndexChange} value={index}>
-            <Tab label="Design" />
-            <Tab label="Scoring" />
+            <Tab label="Design"/>
+            <Tab label="Scoring"/>
           </Tabs>
           <Help />
         </div>
         {index === 0 && (
           <Design
             model={model}
-            onModelChange={this.onModelChange}
+            onModelChanged={this.onModelChanged}
```

Comment:
`updateModel` and `onModelChange` this looks odd, 1. there should be only 1 function that updates the model, 2. try and keep naming consistent, for a callback use `onThingVerb` eg: `onPlaneFlew` `onModelChanged` etc.